README.md

```diff
@@ -19,6 +19,7 @@ Read [Developer Guide](./doc/dev/developer_guide.md) for setting up development
 ## Requirements
 
 - Kubernetes 1.4+
+  **Warning**: as of Kubernetes 1.5.2 this guide won't work due to issue https://github.com/kubernetes/kubernetes/issues/39816 .
```

Comment:
not this guide wont work

etcd operator does not work well with Kubernetes 1.5.2 due to a Kubernetes bug: